server/Arcadia.Assistant.Calendar/SickLeave/EmployeeSickLeaveActor.cs

```diff
@@ -10,17 +10,20 @@
 
     public class EmployeeSickLeaveActor : CalendarEventsStorageBase
     {
-        public EmployeeSickLeaveActor(string employeeId)
+        public EmployeeSickLeaveActor(string employeeId, IActorRef sickLeaveNotifications)
             : base(employeeId)
         {
             this.PersistenceId = $"employee-sickleaves-{this.EmployeeId}";
+            this.sickLeaveNotifications = sickLeaveNotifications;
         }
 
         public override string PersistenceId { get; }
 
-        public static Props CreateProps(string employeeId)
+        private readonly IActorRef sickLeaveNotifications;
```

Comment:
a very minor thing, but, in general, people usually keep private readonly fields on very top of the class, then fields, then properties and constructor, then public methods, then private methods. It's easier to read the class declaration this way.